src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+interface Props {
+  todos: Todo[],
+  removeTodo: (id: number) => void,
+  changeStatus: (id: number, property: Partial<Todo>) => void,
+}
+export const TodosList: FC<Props> = ({ todos, removeTodo, changeStatus }) => {
```

Comment:
The exported component name 'TodosList' does not match the base filename 'TodoList'. Consider renaming the component to 'TodoList'.